src/main/java/org/zalando/jackson/datatype/money/MonetaryAmountSerializer.java

```diff
@@ -24,21 +24,34 @@
 import com.fasterxml.jackson.databind.JsonSerializer;
 import com.fasterxml.jackson.databind.SerializerProvider;
 
+import javax.annotation.Nullable;
 import javax.money.CurrencyUnit;
 import javax.money.MonetaryAmount;
+import javax.money.format.MonetaryAmountFormat;
 import java.io.IOException;
 import java.math.BigDecimal;
 
 public final class MonetaryAmountSerializer extends JsonSerializer<MonetaryAmount> {
 
+    private final MonetaryAmountFormat format;
```

Comment:
The source for `MonetaryAmountFormat` at https://github.com/JavaMoney/jsr354-api/blob/master/src/main/java/javax/money/format/MonetaryAmountFormat.java#L55 says

> Instances of this class are not required to be thread-safe

Addditionally, jackson has some support for configuring locales for serialization (https://fasterxml.github.io/jackson-databind/javadoc/2.2.0/com/fasterxml/jackson/databind/SerializationConfig.html#with%28java.util.Locale%29), which might be nice to reuse instead of having one single locale for the singleton ObjectMapper. I don't see yet how this config might be acessible from a `JsonSerializer`.